app/controllers/clarifying-questions/question-page.ts

```diff
@@ -23,6 +24,9 @@ function getClarifyingQuestionPage(req: Request, res: Response, next: NextFuncti
 function postClarifyingQuestionPage(updateAppealService: UpdateAppealService) {
   return async (req: Request, res: Response, next: NextFunction) => {
     try {
+      if (req.body['saveForLater']) {
```

Comment:
you are not saving answers if the appellant enters one.  Take a look at 'saveForLater.ts' controller and copy that logic.